roles/tpa_single_node/tasks/server/trustd.yml

```diff
@@ -6,6 +6,21 @@
     mode: "0600"
   register: configmap_server_api
 
+- name: Set configmaps for server
+  ansible.builtin.set_fact:
+    server_configmaps: >-
+      {{
+        [
+          tpa_single_node_kube_manifest_dir + '/ConfigMaps/server-api.yaml',
+          tpa_single_node_kube_manifest_dir + '/ConfigMaps/trust-anchor.yaml'
+        ] +
+        (
+          [tpa_single_node_kube_manifest_dir + '/ConfigMaps/s3-trust-anchors.yaml']
+          if s3_trust_anchors_configmap_result is defined and s3_trust_anchors_configmap_result.path | length > 0
+          else []
```

Comment:
What is the purpose of this if else block ?

Reply:
Same as above